src/js/pre-need/config/form.js

```diff
@@ -566,7 +570,30 @@ const formConfig = {
               application: {
                 type: 'object',
                 properties: {
-                  attachments
+                  preneedAttachments: _.merge(preneedAttachments, {
+                    items: {
+                      properties: {
+                        attachmentId: {
+                          'enum': [
+                            '1',
+                            '2',
+                            '3',
+                            // '4',
```

Comment:
Is this planned in the future?

Reply:
No, this is a type that they said we could exclude, since it's called "VA PRENEED FORM," which is sort of redundant when that's the form you're filling out.